lib/database/sql/000020_dial_failure.up.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS dialFailure (
+    id SERIAL PRIMARY KEY,
+    recordId INTEGER NOT NULL,
+    errorType INTEGER NOT NULL,
+    errorMsg TEXT,
+    protocols TEXT NOT NULL,
+    timestamp INTEGER NOT NULL,
+    CONSTRAINT dialFailure_unique UNIQUE (recordId, errorType, protocols, timestamp)
```

Comment:
I think this one is missing from the `test_utils.go` cleanup

Reply:
It seems to work without it but I added it anyways